nomos-blend/message/src/crypto/proofs/quota/tests.rs

```diff
@@ -14,8 +14,8 @@ fn secret_selection_randomness_dst_encoding() {
     // Blend spec: <https://www.notion.so/nomos-tech/Proof-of-Quota-Specification-215261aa09df81d88118ee22205cbafe?source=copy_link#25e261aa09df802d87edfc54d1d60b80>
     assert_eq!(
         *DOMAIN_SEPARATION_TAG_FR,
-        BigUint::from_bytes_be(
-            &<[u8; 23]>::from_hex("0x31565f5353454e4d4f444e41525f4e4f495443454c4553").unwrap()
+        BigUint::from_bytes_le(
```

Comment:
Actually this should also use `fr_from_bytes`, sorry I missed this one.

Reply:
Ah yeah I missed it because we decided not to export BE and LE from the `groth16` crate, so doing BE stuff would involve doing things manually, which is now not the case anymore.

Reply:
Since this is a test, I chose to use `fr_from_bytes_unchecked` in the new commit. Actually, I think we can use it for other part of the code but didn't touch it since I'm not sure if there's anything I'm missing.

Reply:
Mhh the only other part where we could use it is for DSTs and public key split up. Nevertheless, I think it's better to leave `expect`s, and use unchecked only for tests, in my opinion.